lib/deps/clone.js

```diff
@@ -38,6 +38,10 @@ module.exports = function clone(object) {
     return object.toISOString();
   }
 
+  if (object instanceof ArrayBuffer) {
+    return object.slice(0);
+  }
+
```

Comment:
Yeh we have another around this code that I am working on in https://github.com/pouchdb/pouchdb/issues/4273